### PR TITLE
fixes #13 - add read concern

### LINKED ISSUE #13: Read from primary
We have found there is some lag between mongo syncs and how fast the application works. As a work around, add an optional time delay.

eg. .../test?delay=1

### PR DESCRIPTION
**Pull request by dcrissman**

### CODE REVIEW

File: src/main/java/com/redhat/lightblue/healthcheck/HealthCheckResource.java [hunk 1]
```diff
@@ -97,6 +99,7 @@ private Test find(String uuid) throws LightblueException {
         DataFindRequest findRequest = new DataFindRequest(ENTITY);
         findRequest.select(Projection.includeFieldRecursively("*"));
         findRequest.where(Query.withValue("_id", Query.eq, uuid));
+        findRequest.execution(Execution.MongoController.withReadPreference(ReadPreference.primary));
```
Comment (dcrissman) [inline src/main/java/com/redhat/lightblue/healthcheck/HealthCheckResource.java]: @paterczm - is there a way we could shortcut this call? Maybe just Execution.withReadPreference(...);

**Comment by dcrissman** [comment]
@jewzaam - Should I also be using a writeConcern? I think the answer is no, but mostly just to confirm, a write will always be against the primary mongo instance anyway. So if writing and reading to the primary node, we should be good.

**Comment by jewzaam** [comment]
Write concern controls the acknowledgements from the replica set.  The
lightblue crud hosts have a default set and that should be used.
https://docs.mongodb.com/manual/reference/write-concern/

File: src/main/java/com/redhat/lightblue/healthcheck/HealthCheckResource.java (hunk 1 above)
Comment (dcrissman) [inline src/main/java/com/redhat/lightblue/healthcheck/HealthCheckResource.java]: https://github.com/lightblue-platform/lightblue-client/pull/301

**Comment by dcrissman** [comment]
Ready for review

**Pull request merged**